spec/ParseUser.spec.js

```diff
@@ -1784,6 +1784,34 @@ describe('Parse.User testing', () => {
         var b = JSON.parse(body);
         expect(typeof b.sessionToken).toEqual('string');
         expect(typeof b.createdWith).toEqual('object');
+        expect(b.createdWith.action).toEqual('signup');
```

Comment:
it's now signup to match original parse.com behaviour as described [here](https://github.com/ParsePlatform/parse-server/issues/1097#issuecomment-198887783)